.github/actions/contentful-checks/action.yml

```diff
@@ -42,14 +42,23 @@ runs:
         CONTENTFUL_SPACE_ID: ${{ inputs.contentful-space-id }}
         CONTENTFUL_ENV_ID: ${{ inputs.contentful-master-env }}
         CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ inputs.contentful-token }}
+      # Determine whether the Contentful webhook setup has changed since master
+    - uses: technote-space/get-diff-action@v6
+      with:
+        PATTERNS: |
+          +src/**/*.ts
```

Comment:
if we add this pattern isn't going to select all files that match this pattern?

looking at the repo (https://github.com/technote-space/get-diff-action#usage) it seems it would
<img width="911" alt="Screenshot 2023-03-30 at 08 25 58" src="https://user-images.githubusercontent.com/16595804/228821824-b061f4a2-324e-42e4-b50d-fc01ef48978c.png">

and we want to see the changes only in `setup-webhook.ts` right?

Reply:
Also are these paths relative to the project root, or some other subdirectory? It looks like they're intended to be relative to the `./packages/contentful/` directory, but I can't see what code there is that causes that to be the case.